server/notebooks/views.py

```diff
@@ -7,12 +7,17 @@
 from django.urls import reverse
 from django.views.decorators.csrf import ensure_csrf_cookie
 
+from .models import Notebook, NotebookRevision
+from .names import get_random_compound
```

Comment:
If you re-run the latest version of isort against this file, it should fix the issue.

Reply:
Great! Working on some Docker surgery right now, then I'll take care of that. :smiley: